src/rendering/vcTriangulate.cpp

```diff
@@ -147,7 +147,9 @@ bool vcTriangulate_Process(const udDouble2 *pContours, int contourCount, std::ve
 
       /* remove v from remaining polygon */
       for (s = v, t = v + 1;t<nv;s++, t++)
-        V[s] = V[t]; nv--;
+        V[s] = V[t];
+
+      nv--;
```

Comment:
This should have been a separate PR as its very unrelated. I understand why it was required though and the code quality was very low.

Reply:
Yeah, it was due to the update compiler with Ubuntu 18.04, I wasn't sure how many problems were going to pop up.